doc/source/programs/gdal_grid.rst

```diff
@@ -48,6 +48,9 @@ computer.
 
 .. include:: options/ot.rst
 
+   If not set then a default type is used, which might not be supported
+   by the relevant driver.
+
```

Comment:
You cannot put content after a ``.. include`` statement.
I don't see this comment as really necessary as the user will get an explicit error when the situation arises. If you're keen on keeping this comment, you should move it just before the options detail for example, at the topo of the "Description"  paragraph

Reply:
Fixed in #9333.